### PR TITLE
Downgrading Node version from 22.11.0 to 22.5.1 for DigitalOcean

### PR DESCRIPTION
**Pull request by tellmeY18**
## Proposed Changes

The DigitalOcean buildpack supports the following Node.js runtime versions for Ubuntu 22.04:
	•	4.x - 19.x
	•	20.0.0 - 20.9.0
	•	21.0.0 - 21.7.3
	•	22.0.0 - 22.5.1

Node.js v20.11.0 builds fail during deployment. Downgrading to v22.5.1 resolves the issue.

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Expanded compatibility by specifying the Node.js version requirement to allow for newer minor or patch versions of 22.11.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -172,7 +172,7 @@
     ]
   },
   "engines": {
-    "node": ">=22.11.0"
+    "node": ">=22.5.1"
```
[the remark this answers is not in the document]
Comment (rithviknishad) [reply package.json]: Title renamed, was a typo

**Review by rithviknishad — APPROVED** [review]

**Comment by sainak** [comment]
Digital ocean allows using heroku-buildpack-nodejs which supports newer nodejs versions and is more maintained

**Comment by sainak** [comment]
digital ocean is using the latest version of heroku build pack, their documentation is outdated
here is the log of the most recent build
image

we should change the `>=22.11.0` to `^22.11.0` so that the pipelines install the correct major version

**Comment by nihal467** [comment]
https://rebuildearth.slack.com/archives/C010GQBMFJ9/p1733752376504619 Closing based on the conversation

**Pull request closed without merging**